service_capacity_modeling/models/org/netflix/postgres.py

```diff
@@ -0,0 +1,125 @@
+from typing import Any
+from typing import Dict
+from typing import Optional
+from typing import Tuple
+
+from service_capacity_modeling.interface import AccessConsistency, Platform
+from service_capacity_modeling.interface import AccessPattern
+from service_capacity_modeling.interface import CapacityDesires
+from service_capacity_modeling.interface import CapacityPlan
+from service_capacity_modeling.interface import Consistency
+from service_capacity_modeling.interface import DataShape
+from service_capacity_modeling.interface import Drive
+from service_capacity_modeling.interface import FixedInterval
+from service_capacity_modeling.interface import GlobalConsistency
+from service_capacity_modeling.interface import Instance
+from service_capacity_modeling.interface import Interval
+from service_capacity_modeling.interface import QueryPattern
+from service_capacity_modeling.interface import RegionContext
+from service_capacity_modeling.models import CapacityModel
+from . import nflx_aurora_capacity_model, nflx_cockroachdb_capacity_model
+
+
+class NflxPostgresCapacityModel(CapacityModel):
+    @staticmethod
+    def capacity_plan(
+        instance: Instance,
+        drive: Drive,
+        context: RegionContext,
+        desires: CapacityDesires,
+        extra_model_arguments: Dict[str, Any],
+    ) -> Optional[CapacityPlan]:
+        if desires.service_tier == 0:
+            return None
+
+        require_multi_region: bool = extra_model_arguments.get("require_multi_region", False)
```

Comment:
can you change this to infer the default from the number of desire locations?

Reply:
Are we doing regional planning and control number of regions from the provision pipelines?